specs/core/0_beacon-chain.md

```diff
@@ -629,6 +630,16 @@ The `hash` function is SHA256.
 
 `def signing_root(object: SSZContainer) -> Bytes32` is a function defined in the [SimpleSerialize spec](../simple-serialize.md#self-signed-containers) to compute signing messages.
 
+### `bls_domain`
+
+```python
+def bls_domain(domain_type: int, fork_version=b'\x00\x00\x00\x00') -> int:
```

Comment:
1. use `bytes` for type hinting
```suggestion
def bls_domain(domain_type: int, fork_version: bytes=b'\x00\x00\x00\x00') -> int:
```

2. or define `ForkVersion` or `Bytes4` in Custom types section
```
| `ForkVersion` | `bytes4` | 4 bytes of binary data |
```
and then
```suggestion
def bls_domain(domain_type: int, fork_version: ForkVersion=b'\x00\x00\x00\x00') -> int:
```

Reply:
rolled with `bytes` for now